### PR TITLE
Send DUT serial to OTNS

### PR DESCRIPTION
**Pull request by ctan-g**
For future OTNS functionality to display node details, this commit allows OTNS manager to send `dut_serial` events to OTNS.

### CODE REVIEW

File: silk/tools/otns_manager.py [hunk 1]
```diff
@@ -328,6 +332,13 @@ def send_extaddr_event(self):
         self.logger.debug(f"Node {self.node_id} sending extaddr={self.extaddr:016x}")
         self.send_event(event.to_bytes())
 
+    def send_dut_serial_event(self):
+        """Send DUT serial event.
+        """
+        event = Event.status_event(f"dut_serial={self.dut_serial}")
```
Comment (simonlingoogle) [inline silk/tools/otns_manager.py:338]: What is a typical DUT serial look like? 
Do you have an example?

Comment (ctan-g) [reply silk/tools/otns_manager.py:338]: It's usually a 9-digits string for nRF boards.

Comment (qilinkang) [reply silk/tools/otns_manager.py:338]: Defined in hwconfig.ini. Example:
DutSerial: 683536778

**Review by qilinkang — APPROVED** [review]
LGTM

**Review by jwhui — APPROVED** [review]

**Pull request merged**